go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/SafetyCulture/safetyculture-exporter-ui
 go 1.18
 
 require (
-	github.com/SafetyCulture/safetyculture-exporter v0.0.0-20230306044435-b83b54bbdd0d
+	github.com/SafetyCulture/safetyculture-exporter v0.0.0-20230327025354-362d100ba0e5
```

Comment:
probably should use the actual version (v.4.6.0) instead of timestamp and commit hash. thoughts?